.golangci.yml

```diff
@@ -5,7 +5,6 @@ linters:
   disable-all: true
   enable:
     - bodyclose
-    - depguard
```

Comment:
This linter was causing CI errors but we never configured it and probably it just got enabled with a default blocking rule in #2148. cc @robert-zaremba